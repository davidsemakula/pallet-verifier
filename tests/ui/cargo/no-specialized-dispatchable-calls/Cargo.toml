[package]
name = "no-specialized-dispatchable-calls"
version = "0.1.0"
edition = "2021"

[dependencies]
frame-support = "29.0.2"
frame-system = "29.0.0"
parity-scale-codec = "3.6.12"
scale-info = "2.11.3"