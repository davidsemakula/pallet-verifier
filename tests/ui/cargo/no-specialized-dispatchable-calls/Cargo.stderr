error: Failed to generate tractable entry points for any dispatchable function in public interface
   |
   = help: Add some unit tests for dispatchable functions.

error: could not compile `no-specialized-dispatchable-calls` (lib test) due to 1 previous error
